gdk-pixbuf/src/pixbuf.rs

```diff
@@ -344,6 +344,7 @@ impl Pixbuf {
     }
 
     #[cfg(any(feature = "v2_32", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v2_32")))]
```

Comment:
Do you have a script to update all these?

Reply:
No, by hand.